runners/core-java/build.gradle

```diff
@@ -46,6 +46,8 @@ dependencies {
   implementation library.java.joda_time
   implementation library.java.vendored_grpc_1_60_1
   implementation library.java.slf4j_api
+  implementation library.java.jackson_core
+  implementation library.java.jackson_databind
```

Comment:
@robertwb this is unfortunate. Do we have a check for API surface that this does not leak anywhere? In old construction-java and in sdk-core this is shaded. I can shade it, but runners-core is currently "plain", so that could be argument against placing it into the runners-core. I'm not familar enough wth the details of the build system to make a decision myself.

Reply:
Ugh...

@kennknowles would be in a better position to answer this. 

We could re-introduce runners-core-construction with this as the sole class if needed if we can't place it in runners-core itself.

Reply:
The `shadow` configuration apparently means the opposite than I would expect. sdks-java-core already has jackson as a runtime dependecny, thus this only fixes checkstyle to not to complain on undeclared transitive dependency. Should be safe to merge.